packages/Webkul/Shop/src/Resources/views/products/add-to-cart.blade.php

```diff
@@ -4,8 +4,7 @@
     $width = (core()->getConfigData('catalog.products.storefront.buy_now_button_display') == 1) ? '49' : '100';
 @endphp
 
-<button type="submit" class="btn btn-lg btn-primary addtocart" {{ ! $product->isSaleable() ? 'disabled' : '' }}
-style="width: <?php echo $width.'%';?>;">
+<button type="submit" class="btn btn-lg btn-primary addtocart" {{ ! $product->isSaleable() ? 'disabled' : '' }}>
```

Comment:
`addtocart` should be kebab case. `add-to-cart` for proper class name consistency.